app/versioning.py

```diff
@@ -0,0 +1,64 @@
+from typing import Callable, Iterable
+
+import flask
+from werkzeug.exceptions import BadRequest
+
+VALID_API_VERSIONS = ('1.0',)
+"""API versions that are valid for a user to request.
+
+NOTE: the first value in this list should be the latest supported version, but the
```

Comment:
Nit: We should change the wording in this comment to say `tuple` not `list`, but not a blocker.

I'll probably go ahead and fix this myself after it's merged.